ush/templates/coldstart.yaml

```diff
@@ -0,0 +1,156 @@
+  nodes: 4:ppn=12
+  jobname: '&MAKE_ICS_TN;'
+  join: !cycstr {value: '&LOGDIR;/&MAKE_ICS_TN;_@Y@m@d@H.log'}
+  envars:
+    <<: *default_vars
+    SLASH_ENSMEM_SUBDIR: '/'
+  dependency:
+    and: &make_ics_deps
+      taskdep_get_extrn:
+        attrs:
+          task: get_extrn_ics
+      or_grid:
+        datadep_complete:
+          attrs:
+            age: 00:00:00:05
+          text: '&LOGDIR;/&MAKE_GRID_TN;_task_complete.txt'
+        streq:
+          left: '&RUN_TASK_MAKE_GRID;'
+          right: 'FALSE'
+      or_orog:
+        datadep_complete:
+          attrs:
+            age: 00:00:00:05
+          text: '&LOGDIR;/&MAKE_OROG_TN;_task_complete.txt'
+        streq:
+          left: '&RUN_TASK_MAKE_OROG;'
+          right: 'FALSE'
+      or_sfc_climo:
+        datadep_complete:
+          attrs:
+            age: 00:00:00:05
+          text: '&LOGDIR;/&MAKE_SFC_CLIMO_TN;_task_complete.txt'
+        streq:
+          left: '&RUN_TASK_MAKE_SFC_CLIMO;'
+          right: 'FALSE'
+
+task_make_lbcs:
+  <<: *default_task
+  command: '&LOAD_MODULES_RUN_TASK_FP; "&MAKE_LBCS_TN;" "&JOBSDIR;/JREGIONAL_MAKE_LBCS"'
+  nodes: 4:ppn=12
+  jobname: '&MAKE_LBCS_TN;'
+  join: !cycstr {value: '&LOGDIR;/&MAKE_LBCS_TN;_@Y@m@d@H.log'}
+  envars:
+    <<: *default_vars
+    SLASH_ENSMEM_SUBDIR: '/'
+  dependency:
+    and:
+      <<: *make_ics_deps
+      taskdep_get_extrn:
+        attrs:
+          task: get_extrn_lbcs
+
+
+task_run_fcst:
+  <<: *default_task
+  command: '&LOAD_MODULES_RUN_TASK_FP; "&RUN_FCST_TN;" "&JOBSDIR;/JREGIONAL_RUN_FCST"'
+  cores: '12'
+  native: '--cpus-per-task 2 --exclusive'
+  nodes: None
+  walltime: 04:30:00
+  jobname: '&RUN_FCST_TN;'
+  join: !cycstr {value: '&LOGDIR;/&RUN_FCST_TN;_@Y@m@d@H.log'}
+  envars:
+    <<: *default_vars
+    SLASH_ENSMEM_SUBDIR: '/'
+    ENSMEM_INDX: ''
+  dependency:
+    and:
+      taskdep_make_ics:
+        attrs:
+          task: '&MAKE_ICS_TN;'
+      taskdep_make_lbcs:
+        attrs:
+          task: '&MAKE_LBCS_TN;'
+
+
+metatask_run_post:
+  var:
+    fh: '000 001 002 003 004 005 006'
+  task_run_post:
+    <<: *default_task
+    attrs:
+      name: '&RUN_POST_TN;_f#fhr#'
+      cycledef: forcast
+      maxtries: '2'
+    command: '&LOAD_MODULES_RUN_TASK_FP; "&RUN_POST_TN;" "&JOBSDIR;/JREGIONAL_RUN_POST"'
+    nodes: 2:ppn=24
+    walltime: 00:15:00
+    jobname: '&RUN_POST_TN;_f#fhr#'
+    join: !cycstr {value: '&LOGDIR;/&RUN_POST_TN;_f#fhr#_@Y@m@d@H.log'}
+    envars:
+      <<: *default_vars
+      SLASH_ENSMEM_SUBDIR: '/'
+      cyc: !cycstr {value: '@H'}
+      fhr: '#fhr#'
+    dependency:
+      or:
+        taskdep:
+          attrs:
+            task: run_fcst
+        and:
+          datadep_dyn:
+            text: !cycstr {value: '&CYCLE_BASEDIR;/@Y@m@d@H/dynf#fhr#.nc'}
+            attrs:
+              age: '05:00'
+          datadep_phy:
+            text: !cycstr {value: '&CYCLE_BASEDIR;/@Y@m@d@H/phyf#fhr#.nc'}
+            attrs:
+              age: '05:00'
```

Comment:
Is the second part of `datadep_dyn/phys/complete` etc parsed and used ? I am assuming reason why it is not just `datadep` is for the readablity of the yaml file?

Reply:
It is _not_ used. It's partially for readability, but also because you can't have repeated keys in the dict.